app/services/activity_logger.py

```diff
@@ -82,6 +96,34 @@ async def log_activity(integration_id: str, action_id: str, title: str, level="I
     )
 
 
+async def log_webhook_activity(
+        integration_id: str, title: str, webhook_id: str="webhook", level="INFO", config_data: dict = None, data: dict = None
+):
+    """
+        This is a helper method to send custom activity logs to the portal.
+        :param integration_id: UUID of the integration
+        :param title: A human-readable string that will appear in the activity log
+        :param webhook_id: str id of the webhook being executed
+        :param level: The level of the log, e.g. DEBUG, INFO, WARNING, ERROR
+        :param data: Any extra data to be logged as a dict
+        :return: None
+        """
+    logger.debug(f"Logging custom activity: {title}. Integration: {integration_id}. Webhook: {webhook_id}.")
+    await publish_event(
+        event=IntegrationWebhookCustomLog(
+            payload=CustomWebhookLog(
+                integration_id=integration_id,
+                webhook_id=webhook_id,
+                config_data=config_data or {},
```

Comment:
@marianobrc is there any chance that `config_data` has sensitive values?

Reply:
do you think it's better to remove this cofig_data from logs in both webhooks and actions for now @chrisdoehring?

Reply:
Ah of course. That makes sense and I agree it's good to leave the config_data in the logs. _Apologies for the slow reply._